helix-loader/src/grammar.rs

```diff
@@ -127,18 +127,14 @@ where
             tx.send(job(grammar)).unwrap();
         });
     }
-    pool.join();
+
+    drop(tx);
```

Comment:
I'm not very knowledgeable about the pool or mpsc stuff - most of that was migrated from helix-syntax. Based on the docs this looks good to me though :+1: